app/models/tag.rb

```diff
@@ -132,8 +133,10 @@ def find_or_create_by_names(name_or_names)
 
     def search_for(term, limit = 5, offset = 0, options = {})
       stripped_term = term.strip
+      options = { exclude_unlistable: true, exclude_unreviewed: false }.merge(options)
```

Comment:
I could've used `merge!` here, but that felt kind of wrong for some reason

Reply:
That is one reason `ActiveSupport` adds `#reverse_merge`. So you could do:

```ruby
options.reverse_merge!({ exclude_unlistable: true, exclude_unreviewed: false })
```

(I just wanted to provide an alternative, but do not have a strong preference myself.)